samples/sensor/bme68x/README.rst

```diff
@@ -0,0 +1,48 @@
+.. _bme68x:
+
+Gas Sensor BME688
+###############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample application sets up the BME68X gas sensor with the Bosch Sensor Environmental Cluster (BSEC) library.
+
+Requirements
+************
+
+Using this driver requires manually downloading the BSEC library. See :ref:`lib_bme68x` for more details.
+The sample supports the following devices:
+
+.. table-from-sample-yaml::
+
+Building and running
+********************
+
+This project outputs sensor data to the console.
+It requires a BME68X sensor.
+Board overlays for Thingy:91 and Thingy:53 are included.
+
+.. |sample path| replace:: :file:`samples/sensor/bme68x`
+
+.. include:: /includes/build_and_run.txt
+
+Sample output
+=============
+
+The following output is displayed in the terminal:
+
+.. code-block:: console
+
+    *** Booting Zephyr OS build v3.2.99-ncs1-1531-gaf18f6b63608 ***
+    [00:00:01.285,339] <inf> app: App started
+    [00:00:07.287,658] <inf> app: temp: 28.240385; press: 100043.570312; humidity: 19.981348; iaq: 50
+    [00:00:10.301,391] <inf> app: temp: 28.220613; press: 100039.585937; humidity: 19.983814; iaq: 50
+    [00:00:13.315,124] <inf> app: temp: 28.188013; press: 100040.007812; humidity: 20.015941; iaq: 50
```

Comment:
Should it mention in the sample that the iaq value takes ~24 hours to auto-calibrate?